bindings/python/pymongoarrow/bson_util.pyx

```diff
@@ -0,0 +1,33 @@
+# distutils: language=c++
+# cython: language_level=3
+
+from libcpp cimport bool as cbool
+
+from pymongoarrow.libbson cimport *
+
+
+cdef list _docs_from_bson_stream(const uint8_t *docstream, size_t length):
+    cdef bson_reader_t *stream_reader = bson_reader_new_from_data(docstream, length)
```

Comment:
Does this need a NULL check?

Reply:
`bson_util` has been removed since it wasn't intended to be a part of the final package and only served as an example.

Reply:
Ack.